app/models/host/managed.rb

```diff
@@ -912,22 +912,6 @@ def recreate_config(only = nil)
     result
   end
 
-  # converts a name into ip address using DNS.
-  # if we are managing DNS, we can query the correct DNS server
-  # otherwise, use normal systems dns settings to resolv
-  def to_ip_address(name_or_ip)
```

Comment:
@tbrisker : Thanks for reviewing. I don't think, this is necessary. But it's easily done. Let me know and I'll get started.